validator/project_validator.go

```diff
@@ -174,16 +174,32 @@ func CheckProjectConfigurationIsValid(project *model.Project) error {
 		}
 	}
 	if len(syntaxErrs) > 0 {
-		return gimlet.ErrorResponse{
-			StatusCode: http.StatusBadRequest,
-			Message:    fmt.Sprintf("project syntax is invalid: %s", ValidationErrorsToString(syntaxErrs)),
+		syntaxErrsAtErrorLevel := ValidationErrors{}
```

Comment:
Aren't all the "syntax" validators errors, and the "semantics" validators warnings? So you can check one but not the other?